fio.run-file/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# user configurable options #####################################################
+
+# remotehost to test
+remote_host=foo.example.com
+
+# set the number of samples for your run
+samples=3
+
+# userenv to build containers using
+userenv="rhubi8"
```

Comment:
What is the use case of these replacements ? I thought the idea was to promote usage of crucible run directly from command line without any scripts. The helper script to replace is good, but could be separated from the normal use.

Reply:
The `run.sh` just serves as an example of how the user changes some values in the JSON file and then invokes the run command.  One of the values, which uses the variable `remote_host`, must be changed by the user -- it is impossible for us to put a reasonable default in the JSON file unless we decided to use `localhost` and promote running in "loopback" mode.  The other values, `samples` and `userenv`, are just randomly chosen to given the user additional insight into how they might tweak the run to change it's behavior.  These happen to be the same variables from the existing fio example.

A user is more than welcome to run from the CLI directly, but there since it's almost impossible for us to ship an example that doesn't require any modification to be able to run, see above, the `run.sh` just serves the purpose of showing the basic steps and requirements necessary to use Crucible with run-file.

Reply:
Ok, this is more a usability opinion that I have: perhaps my point of view is not what most of users will think then...